### PR TITLE
Sign up

### PR DESCRIPTION
**Pull request by frmad**
I have added a signup screen, made the links and made some improvements on the settings screens (login, sign up, settings)

### CODE REVIEW

File: src/utility/authentication.ts [hunk 1]
```diff
@@ -45,8 +46,36 @@ export const useAuthenticationStore = create(persist<AuthenticationStore>(
...
+                }).toString(),
+            });
+            if (response.ok) {
+                const response_data = await response.json()
+                set({ state: AuthenticationState.Authenticated, token: response_data.access_token })
+            } else {
+                set({ state: AuthenticationState.Error })
+            }
             queryClient.invalidateQueries()
+            {/*if (response.ok) {
```
Comment (Nolle10) [inline src/utility/authentication.ts]: Please remove these comments before submitting it

**Review by Nolle10 — CHANGES_REQUESTED** [review]
Please remove unnecessary comments before submitting

**Review by Darchie4 — APPROVED** [review]
If it works, it LGTM

**Review by Nolle10 — APPROVED** [review]
I approve

**Pull request merged**